README.md

```diff
@@ -32,6 +32,8 @@ After adding the dependency using composer (as described above) to your applicat
 
 Copy the folders (and JSON files) of languages that you want to use, into the directory `resources/lang` of your Laravel application (or `app/lang` in Laravel 4).
 
+If you are using Laravel Mix, add `mix.copy('vendor/laravel-lang/lang/src/ms', 'resources/lang/ms');` to your `webpack.mix.js` to let mix automatically copy the directory every time you compile.
```

Comment:
You can't do that. In translation files, the developer can indicate his own keys. The `mix.copy` command will remove them, replacing them with clean files from this project.
In addition, you need to manually specify which localizations to copy.
The correct way is to use a translation manager. For example, [this one](https://github.com/andrey-helldar/laravel-lang-publisher).